config.py

```diff
@@ -0,0 +1,111 @@
+import os
+import types
+from functools import reduce
+
+_GLOBAL_CONFIG = dict(
+    # If running on Tools labs, keep database dumps in this directory...
+    archive_dir = os.path.join(os.path.expanduser('~'), 'archives'),
+
+    # ...and delete dumps that are older than this many days
+    archive_duration_days = 90,
+
+    # Where to put various logs
+    log_dir = os.path.join(os.path.expanduser('~'), 'logs'),
+
+    flagged_off = [],
+
+    profile = True,
+
+    stats_max_age_days = 90,
```

Comment:
I think none of the config options up to here actually apply to us now -- they're for controlling backups, profiling and stats (https://tools.wmflabs.org/citationhunt/stats.html).

Reply:
Oh ok! I’ll remove them. I was not sure what are these for.